lib/src/rewrite.rs

```diff
@@ -241,9 +241,21 @@ pub enum EmptyBehaviour {
 // change the RebaseOptions construction in the CLI, and changing the
 // rebase_commit function to actually use the flag, and ensure we don't need to
 // plumb it in.
-#[derive(Clone, Default, PartialEq, Eq, Debug)]
+#[derive(Clone, PartialEq, Eq, Debug)]
 pub struct RebaseOptions {
     pub empty: EmptyBehaviour,
+    /// If a merge commit would end up with one parent being an ancestor of the
+    /// other, then filter out the ancestor.
+    pub no_ancestor_merge: bool,
```

Comment:
nit: I prefer `simplify_ancestor_merge` or something like that. `no_` sounds like a flag to disable extra operation.

Reply:
Done.